### PR TITLE
Feature/middle section works about me

### PR DESCRIPTION
**Pull request by Braingenius2**
# Add middle sections to portfolio site: Works and About me
In this pull request, I

- Built Works section using CSS grid and flexbox layout tools
- Built About me section using a flexbox layout
- Added additional background images in the About me section.
- Implemented the button interactions (enable, hover, etc.).

### CODE REVIEW

**Review by Trast00 — COMMENTED** [review]
Hi @Braingenius2 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that have failing linter errors. Please fix the linter before ask for a review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@Trast00) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 1]
```diff
@@ -52,6 +52,175 @@ <h1>
             </ul>
             <div class="disabled"><img src="images/disabled.png" alt=""></div>
         </section>
+
+        <section id="works">
```
Comment (divinecharlotte) [inline index.html:56]: - [ ] i can see that you used flexbox to align this sections content thats also good but its not meeting  the below requirement
> The main container of the projects has a display:grid
kindly consider changing the `display:flex` to `display:grid ` to full-fill that requirement

File: index.html [hunk 2]
```diff
@@ -52,6 +52,175 @@ <h1>
...
+                        <figcaption></figcaption>
+                    </figure>
+                    <div class="text">
+                        <h2>Multi-Post Stories <br>Gain+Glory</h2>
+                        <ul class="tags">
+                            <li>Ruby on Rails</li>
+                            <li>CSS</li>
+                            <li>JavaScript</li>
+                            <li>HTML</li>
+                        </ul>
```
Comment (divinecharlotte) [inline index.html:90]: - [ ] kindly consider making sure that you stick to the figma design as much as possible please put all there li's on the same line to have your card look like the one in the figma design 
- your design 👇 
Screen Shot 2023-04-01 at 21 21 10


- given design 👇 
Screen Shot 2023-04-01 at 21 21 47

File: index.html [hunk 3]
```diff
@@ -52,6 +52,175 @@ <h1>
...
+                <h1>About me</h1>
+                <p>I can help you build a product, 
+                    <span> feature or website Look through</span> 
+                    <span> some of my work and experience! If</span>
+                    <span> you like what you see and have a </span>
+                    <span> project you need coded,</span> 
+                    <br> don't hesitate to contact me.</p>
+                <button class="get-resume" type="button">Get my resume</button>   
+
+            </div>
```
Comment (divinecharlotte) [inline index.html:172]: - [ ] good implementation of this div but it still missing something which is the background image kindly consider adding that to implement exactly this section as expected

File: index.html [hunk 4]
```diff
@@ -52,6 +52,175 @@ <h1>
...
+                        <figcaption></figcaption>
+                    </figure>
+                    <div class="content">
+                        <h2>Skills</h2>
+                        <ul class="tags">
+                            <li>Database Management</li>
+                            <li>Version Control</li>
+                            <li>CLI</li>
+                            <li>Web Development</li>
+                            <li>API Design</li>
```
Comment (divinecharlotte) [inline index.html:217]: - [ ] kindly consider adding the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li` to fullfill the below requirement 👇 
- > The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @Braingenius2       , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 


- Descriptive PR ✔️
- linters are passing 👍  

## Required Changes ♻️

- [ ] in your readme file kindly consider adding working links for the following : your Linkdhin account in authors section; real issue page on the contributing section
 to give a god user experience to the person visiting your readme

_Check the comments under the review._



## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -52,6 +52,175 @@ <h1>
             </ul>
             <div class="disabled"><img src="images/disabled.png" alt=""></div>
         </section>
+
+        <section id="works">
+            <div class="text">
+                <h1><span>My Recent</span> <span>Works</span></h1>
+                <hr class="indicator">
+            </div>
+            <div class="card-work-row">
```
Comment (Braingenius2) [inline index.html:61]: @divinecharlotte If you check well you'd see I used `display: grid ` in the container card-work-row containing all the cards which is the actual core requirement of the project. 
image

File: index.html [hunk 6]
```diff
@@ -52,6 +52,174 @@ <h1>
...
+
+        <section id="about">
+            <div class="text">
+                <h1>About me</h1>
+                <p>I can help you build a product, 
+                    <span> feature or website Look through</span> 
+                    <span> some of my work and experience! If</span>
+                    <span> you like what you see and have a </span>
+                    <span> project you need coded,</span><br> don't hesitate to contact me.</p>
+                <button class="get-resume" type="button">Get my resume</button>
```
Comment (Shedrack-Sunday) [inline index.html:169]: - [ ] As stated by the previous reviewer, you're missing the background images of the about section. I've made detailed comparison below for you to see. This is important to stay t user requirements, as would be so in the future with clients, and give a better UX.
- Please place them appropriately 

## Your design:
image

## Figma:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @KALUNGI-J-BOBSON        👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlight
✔️ good PR summary
✔️ Moderate UI
  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, the changes required below need to be implemented :
- A few styling adjustments are needed👏

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by didierganthier — APPROVED** [review]
Hi @Braingenius2 ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉

**Pull request merged**